specification/communication/data-plane/SipRouting/preview/2024-11-15-preview/communicationservicessiprouting.json

```diff
@@ -166,8 +166,7 @@
     "Trunk": {
       "description": "Represents a SIP trunk for routing calls. See RFC 4904. Can be expanded with additional data.",
       "required": [
-        "sipSignalingPort",
-        "enabled"
+        "sipSignalingPort"
```

Comment:
Why are we removing `enabled` here?

Reply:
It was incorrectly set as required in previous preview. We GA'd Trunk only with sipSignalingPort, later added enabled and incorrectly marked it as required. As per docs, we shouldn't add required properties to avoid breaking change.